cns/service/main.go

```diff
@@ -813,6 +813,11 @@ func main() {
 		if platform.HasMellanoxAdapter() {
 			go platform.MonitorAndSetMellanoxRegKeyPriorityVLANTag(rootCtx, cnsconfig.MellanoxMonitorIntervalSecs)
 		}
+
+		// if swiftv2 scenario is enabled, we need to initialize the ServiceFabric(standalone) swiftv2 middleware to process IPConfigsRequests
+		// we don't need a cns-config check here since this middleware will be called only by CNI for SwiftV2 flow
+		swiftV2Middleware := &middlewares.StandaloneSWIFTv2Middleware{}
+		httpRemoteRestService.AttachIPConfigsHandlerMiddleware(swiftV2Middleware)
```

Comment:
move closer to place where k8s middleware is intialized?

Reply:
i dont see if this is valid outside the "direct channel" block, K8s middleware is initialized inside CRD channel. currently this looks fine in this section, we can address this later when Jackie takes up the PR?